test/test_metrics_converter.py

```diff
@@ -117,3 +119,42 @@ def test_convert_to_metrics_with_metric_dimension():
     metrics = convert_to_metrics(data, dataset, "_")
 
     assert metrics == data.metrics_str("_")
+
+
+class TestParseStatsDSignalFxMetricName:
```

Comment:
❤️ Love the tests - they're clean, easy to read and nicely conveying the purpose of the functionality.

Reply:
+1